### PR TITLE
LPD-15724 Remove ContentPageWithViewport

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:playwright

File: modules/test/playwright/tests/layout-content-page-editor-web/viewport.spec.ts [hunk 1]
```diff
@@ -59,6 +62,28 @@ const NON_DESKTOP_PANELS: NonDesktopPanels = [
...
 ];
 
+const createPageWithFragmentAndGoToEditMode = async ({
+	apiHelpers,
+	fragment,
+	page,
+	pageEditorPage,
+	site,
+}) => {
+	await page.goto(liferayConfig.environment.baseUrl);
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/test/playwright/tests/layout-content-page-editor-web/viewport.spec.ts:72]: By the way, regarding this method, this morning I was precisily discussing with Ivan about this kind of abstractions. I'm usually very fan of them, but it's true that sometimes, it's better to explicitly do things in the test. He told me there are theories that defend that specially for tests, it's convenient to do things explicitly on each test, and limit the abstractions. In this specific case, by doing the abstraction, we are saving just one sentence on each test, and creating a "complex" method that needs 5 params, so I think it's not worth it, what do you think?

**Review by sandrodw3 — COMMENTED** [review]
@veroglez this is super cool 🌈 I left some suggestions, let me know what you think!

**Comment by sandrodw3** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/148081

**Pull request closed without merging**